README.md

```diff
@@ -1,9 +1,15 @@
 # Alpine ROS Docker Images
 Docker image of ROS (Robot Operating System) on Alpine Linux
 
+## News
+
+### April 2023
+`melodic` (Melodic EOL on May 2023) and `noetic-3.11` (Alpine 3.11 EOL on May 2020) is dropped.
+These images are still available but no longer updated. It is highly recommended to update to `noetic-3.14` or `noetic-3.17`.
```

Comment:
[optional] Does it mean we could also remove the `melodic` and `noetic` folders? (and `kinetic` as well).

Reply:
Images should be remained after EOL to make old things reproducible and I think Dockerfiles are also better to be remained.